custom_components/truenas/__init__.py

```diff
@@ -76,6 +82,7 @@ async def async_update_data():
                         machine.get_disks(include_temperature=True),
                         machine.get_jails(),
                         machine.get_vms(),
+                        machine.get_pools(),
```

Comment:
can you move this up a line?  I'm trying to keep this alphabetized so it is easier to find things as this list grows.